HomeTask11/Untitled.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+from sympy import *
+
+
+# In[ ]:
+
+
+
+
+
+# In[2]:
+
+
+x = Symbol('x')
+
+
+# In[3]:
+
+
+fx = x**3 - 50*x
+gx = -x**4 + 88*x**2 - 241
+
+
+# In[4]:
+
+
+inters = solve (fx - gx, x)
+
+
+# In[5]:
+
+
+inters_x = []
+
+
+# In[6]:
+
+
+for i in inters:
+    z = round(i.n(10), 2)
+    inters_x.append(z)
+
+
+# In[7]:
+
+
+inters_x
```

Comment:
В блокноте Jupyter тут бы вывелось что-то но у вас же не блокнот
```suggestion
print(inters_x)
```